src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -174,3 +179,11 @@ def load_command_table(self, _):
 
     with self.command_group('spring-cloud', exception_handler=handle_asc_exception):
         pass
+
+    with self.command_group('spring-cloud build-service builder',
+                            custom_command_type=builder_cmd_group,
+                            exception_handler=handle_asc_exception, is_preview=True) as g:
+        g.custom_command('create', 'create_or_update_builder', supports_no_wait=True)
+        g.custom_command('update', 'create_or_update_builder', supports_no_wait=True)
+        g.custom_command('show', 'builder_show')
+        g.custom_command('delete', 'builder_delete', supports_no_wait=True)
```

Comment:
Do we need to consider adding confirmation=True for the delete operation ?

Reply:
OK.